.coderabbit.yaml

```
@@ -17,6 +26,9 @@ reviews:
   
   # Review entire files for context
   review_full_file: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)